firmware/controllers/serial/serial_tx.cpp

```diff
@@ -0,0 +1,32 @@
+/**
+ * @file	serial_tx.cpp
+ *
+ * This file handles auxilery serial communication.
+ *
+ * @date Mar 19, 2020
+ * @author Konstantin Smola, (c) 2020
+ */
+
+#include "globalaccess.h"
+
+#if EFI_AUX_SERIAL
+#include "engine.h"
+#include "serial.h"
+#include "serial_sensor.h"
+
+#include "allsensors.h"
+#include "vehicle_speed.h"
+
+
+SerialWrite::SerialWrite()
```

Comment:
I am sorry I am failing to comprehend why do we need SerialWrite? Please document the reasons for this in a code comment.

Reply:
It's unused right now, but I figured it might be useful for other sensors/applications in the feature, so the infrastructure is here in case it's needed. I can remove it if you want

Reply:
"useful in the future" is a very slippery slope (don't tell @mck1117 I said that)

please remove this.

Reply:
Do you want me to get rid of the entire file `serial_tx.cpp`?

Reply:
Yes, I believe there is no need in it's current (empty) form